engine/userguide/storagedriver/selectadriver.md

```diff
@@ -33,20 +30,22 @@ this decision, there are three high-level factors to consider:
     and stability in the most usual scenarios.
 
     - `overlay2` is preferred, followed by `overlay`. Neither of these requires
-      extra configuration.
+      extra configuration. `overlay2` is the default choice for Docker CE.
 
     - `devicemapper` is next, but requires `direct-lvm` for production
       environments, because `loopback-lvm`, while zero-configuration, has very
       poor performance.
 
-  The selection order is defined in Docker's source code. You can see the order
-  for Docker 17.03 by looking at
-  [the source code](https://github.com/moby/moby/blob/v17.03.1-ce/daemon/graphdriver/driver_linux.go#L54-L63).
-  For a different Docker version, change the URL to that version.
+  The selection order is defined in Moby's source code. You can see the order
+  by looking at
+  [the source code](https://github.com/docker/docker-ce/blob/master/components/engine/daemon/graphdriver/driver_linux.go#L54-L63).
```

Comment:
Might be good to change this link to point to a specific commit or tag, as "master" can change at any time